tests/wpt/mozilla/tests/css/input_alignment_a.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html>
+<head>
+    <title>Input elements shouldn't inherit parent containers alignment</title>
+    <link rel="match" href="input_alignment_ref.html">
+    <style>
+        div { 
+            text-align: center; 
+            float: left; 
```

Comment:
When we don't add the `float: left;`, parent's `text-align: center;` property centering the whole input element and giving false negative result in reftest. I added float left to this file and the ref file to prevent that. It can be done by giving `display: block;` to input or another better solution would be good :)